x-pack/plugins/cases/server/telemetry/queries/case_system_action.ts

```diff
@@ -26,6 +26,7 @@ export const getCasesSystemActionData = async ({
         cardinality: { field: `${CASE_RULES_SAVED_OBJECT}.attributes.rules.id` },
       },
     },
+    namespaces: ['*'],
```

Comment:
nit: maybe we could also update `case_system_action.test.ts` to have `expect(savedObjectsClient.find...).toHaveBeenCalledWith` or something like that. I see the other tests do that.

Reply:
Good point!